tests/installation/start_install.pm

```diff
@@ -85,7 +85,7 @@ sub run {
         assert_screen("inst-packageinstallationstarted", $started_timeout);
     }
     else {
-        sleep 2;    # textmode is sometimes pressing alt-i too early
+        wait_still_screen;    # textmode is sometimes pressing alt-i too early
```

Comment:
Looks like it's another drawback of disabling grub timeout. But in installation_overview we have assert_screen "installation-settings-overview-loaded", and this needle verifies that Next button is enabled, which means that overview is loaded and button can be pressed. It works well both for textmode and x11, if needle is properly created =)

Reply:
OK, using `assert_screen` instead. Current needles seems to be fine with the thin frame around `Install` button :D.

Reply:
I'm terribly sorry, but I've missed that @okurz has already added such assertion in the end of disable_grub_timeout: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/installation/disable_grub_timeout.pm#L67 So if you have used the same code for 2 runs mentioned in the description, it won't help here or at least won't make it any different =(

Reply:
Np, at all.